skywalking/loggings.py

```diff
@@ -20,8 +20,19 @@
 from skywalking import config
 
 
+def getLogger(name=None):
+    logger = logging.getLogger(name)
+    ch = logging.StreamHandler()
+    formatter = logging.Formatter('%(name)-32s [%(threadName)-15s] [%(levelname)-8s] %(message)s')
+    ch.setFormatter(formatter)
+    logger.addHandler(ch)
+
+    return logger
+
+
+logger = getLogger('skywalking')
+
+
 def init():
-    logging.basicConfig(
-        level=logging.getLevelName(config.logging_level),
-        format='%(name)-32s [%(threadName)-15s] [%(levelname)-8s] %(message)s',
-    )
+    logging.addLevelName(logging.getLevelName('CRITICAL') + 10, 'OFF')
```

Comment:
```suggestion
    logging.addLevelName(logging.CRITICAL + 10, 'OFF')
```

Reply:
Sure, if you want to do it yourself go for it, if not I will do it tomorrow.